ExcelMapper/ExcelMapper.cs

```diff
@@ -214,7 +216,17 @@ IEnumerable<T> Fetch<T>(ISheet sheet) where T : new()
                     {
                         var cell = row.GetCell(col.Key);
                         if (cell != null)
-                            col.Value.SetProperty(o, GetCellValue(cell, col.Value));
+                        {
+                            var cellValue = GetCellValue(cell, col.Value);
+                            try
+                            {
+                                col.Value.SetProperty(o, cellValue);
+                            }
+                            catch (Exception e)
+                            {
+                                throw new ExcelMapperConvertException($"Unable to convert \"{(string.IsNullOrWhiteSpace(cellValue.ToString()) ? "<EMPTY>" : cellValue)}\" from [L:{i}]:[C:{col.Key}] to {col.Value.PropertyType}", e);
```

Comment:
Message is missing punctuation at the end 😉

Reply:
Here you go ! But this will "hurt" test coverage score. Since we're adding 5 constructors but only using one for now and I don't think it's worth to write tests to test exceptions constructors.